url.bs

```diff
@@ -3090,6 +3089,65 @@ e.g., "newURL" and "oldURL".
 naming. [[!HTML]]
 
 
+<h3 id=urlhost-class>URLHost class</h3>
+
+<pre class=idl>
+[Constructor(USVString host),
+ Exposed=(Window,Worker)]
+interface URLHost {
+  USVString unicode();
+  readonly attribute URLHostType type;
+
+  stringifier USVString toJSON();
+};
+
+enum URLHostType { "ipv4", "ipv6", "domain" };</pre>
+
+<p>Each {{URLHost}} object has an associated <dfn for=URLHost>host</dfn> (a <a for=/>host</a>).
+
+<hr>
+
+<p>The <dfn constructor for=URLHost><code>URLHost(<var>input</var>)</code></dfn> constructor, when
+invoked, must run these steps:
+
+<ol>
+ <li><p>Let <var>host</var> be the result of <a>host parsing</a> <var>input</var> with true.
+
+ <li><p>If <var>host</var> is failure, then <a>throw</a> a {{TypeError}}.
+
+ <li><p>Return a new {{URLHost}} object whose <a for=URLHost>host</a> is <var>host</var>.
+</ol>
+
+<p>The <dfn method for=URLHost><code>unicode()</code></dfn> method, when invoked, must run these
+steps:
+
+<ol>
+ <li><p>If <a>context object</a>'s <a for=URLHost>host</a> is not a <a>domain</a>, then <a>throw</a>
+ a {{TypeError}}.
+
+ <li><p>Let <var>unicodeDomain</var> be the result of running <a>domain to Unicode</a> on
+ <a>context object</a>'s <a for=URLHost>host</a>.
+
+ <li><p>Return <var>unicodeDomain</var>, <a lt="host serializer">serialized</a>.
+</ol>
+
+<p>The <dfn attribute for=URLHost><code>type</code></dfn> attribute's getter, must run these steps:
```

Comment:
Either add "when invoked" or be more consistent with other getter descriptions in the standard by removing the awkward comma.